gs-rest-client/pom.xml

```diff
@@ -62,6 +62,8 @@
 
     <properties>
       <java.version>1.8</java.version>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
These changes were necessary to implement switch-case statements, as only newer versions of java support string comparison inside